migrations/1700653450166-set-contextualfilter-to-dashboard-level.ts

```diff
@@ -0,0 +1,98 @@
+import { startDatabaseForMigration } from '@utils/migrations/database.helper';
+import { Application, Dashboard, Page, Workflow } from '@models';
+import { logger } from '@services/logger.service';
+import { contentType } from '@const/enumTypes';
+
+/**
+ * Update dashboard if there is a contextual filter in application level, update it to dashboard level.
+ *
+ * @param dashboard dashboard to update
+ * @param application application to update
+ */
+const updateDashboard = async (
+  dashboard: Dashboard,
+  application: Application
+) => {
+  if (!dashboard.filter) {
+    dashboard.filter = {};
+  }
+  dashboard.filter.structure = application.contextualFilter;
+  dashboard.filter.position = application.contextualFilterPosition;
+
+  await Dashboard.findByIdAndUpdate(dashboard.id, {
+    filter: {
+      ...dashboard.filter,
+      structure: dashboard.filter.structure,
+      position: dashboard.filter.position,
+    },
+  });
+
+  logger.info(
+    `[${application.name} / ${dashboard.name}]: updated filter of application level to dashboard level.`
+  );
+};
+
+/**
+ * Sample function of up migration
+ *
+ * @returns just migrate data.
+ */
+export const up = async () => {
+  await startDatabaseForMigration();
+  const applications = await Application.find()
+    .populate({
+      path: 'pages',
+      model: 'Page',
+    })
+    .select('name pages');
```

Comment:
@estelafs @guisoares1 
I'll do it during the review, but be aware of select in mongoose
there, you don't get the contextualFilter from the application, as it's not part of the list
so you're sure that the dashboard filter will be empty for all dashboards